ffv1.md

```diff
@@ -808,10 +816,26 @@ If state_transition_delta is not present in the FFV1 bitstream, all Range coder
 |-------|:--------------------------------|:--------------------------------|:--------------------------------|
 | 0     | YCbCr                           | No Pixel transformation         | plane then line                 |
 | 1     | RGB                             | JPEG2000-RCT                    | line then plane                 |
+| 2     | Color Filter Array              | JPEG2000-RCT + G difference     | line then plane                 |
 | Other | reserved for future use         | reserved for future use         | reserved for future use         |
 
 Restrictions:  
 If `colorspace_type` is 1, then `chroma_planes` MUST be 1, `log2_h_chroma_subsample` MUST be 0, and `log2_v_chroma_subsample` MUST be 0.  
+If `colorspace_type` is 2, then `chroma_planes` MUST be 1, `log2_h_chroma_subsample` MUST be 0, and `log2_v_chroma_subsample` MUST be 0, transparency MUST be 1.  
```

Comment:
instead of `transparency MUST be 1` you should use the defined name of this value which is `alpha_plane`. But this is a hack, since alpha_plane=1 implies that a transparency plane is present. You'd have to adjust the definition of alpha_plane (maybe rename it), so that you're not implying that there is a transparency plane when there isn't.

Reply:
also should consider https://github.com/FFmpeg/FFV1/issues/47#issue-199430716, which comments on the semantics of infrared scans stored in an alpha plane.

Reply:
Here I "copied" the way RGB support is added, i.e. I keep previous spec as much as possible. I prepare another PR focused on this renaming, which should be merged before this one.